inc/tinyMce.php

```diff
@@ -1,17 +1,9 @@
 <?php
 
 /**
- * Cleans up TinyMCE Buttons to show all relevant buttons on the first bar. Adds an easy to configure way to change these defaults.
+ * Moves most relevant editor buttons to the first toolbar
+ * & provides config for creating new toolbars, block formats, and style formats.
```

Comment:
Full word 'and' would be preferable, same as at the end of sentence.

Reply:
Should we mention and link to docs for further info?

Reply:
Yeah for sure, I just added this in.